packages/web-components/src/components/va-alert/va-alert.scss

```diff
@@ -40,6 +40,10 @@
   padding-right: 3.563rem;
 }
 
+.usa-alert .usa-alert__body {
+  margin-left: 0;
```

Comment:
I'm curious why do we need to override the USWDS module? In the original ticket, Barb mentioned that the [USWDS alert component ](https://designsystem.digital.gov/components/alert/)doesn't do it. But we should just be taking their styles at this point.